catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/federation/impl/SortedFederationStrategy.java

```diff
@@ -127,97 +88,43 @@
    */
   protected List<PostFederatedQueryPlugin> postQuery;
 
-  private SortedQueryMonitorFactory sortedQueryMonitorFactory = new SortedQueryMonitorFactory(this);
+  private SortedQueryMonitorFactory sortedQueryMonitorFactory = new SortedQueryMonitorFactory();
```

Comment:
It is replaced with a mock during testing.  I added a `@VisibleForTesting` to its setter.

Reply:
❓  Any reason it couldn't just get assigned through the constructor instead? Does it need to change mid-test or would re-building the object before each test be unnecessarily verbose?

Reply:
Nope.  Looks like it is only used in the setup method.  I think I was just trying to avoid anything that would impact blueprint but it appears this will only impact the test.  I can fix this one.